@lingosta/app/components/Layout/Navbar.tsx

```diff
@@ -0,0 +1,39 @@
+'use client'
+
+import React from 'react'
+import useUser from '../../hooks/useUser'
+import Link from 'next/link';
+
+const Navbar = () => {
+
+    const { user, logout } = useUser();
+
+    return (
+        <div className='border-b-[1px] border-neutral-200 p-5'>
+            <div className='flex justify-between'>
+                <Link href='/' className='text-lg font-semibold'>
+                    Lingosta
+                </Link>
+                <div>
+                    { !user && (
+                        <div>
+                            <Link href='/login'>
+                                Login
+                            </Link>
+                        </div>
+                    )}
+                    { user && (
```

Comment:
Also, I don't think was showing up when I tested it. I tried logging in and it still prompted me about logging in.

Reply:
I logged in and it was successful in the network tab for the Appwrite API call
```
url: http://localhost:3000/loginStatus: 200 OKSource: NetworkAddress: ::1:3000
```
so something is going on here where login is not working.

Reply:
im not too sure about this one, if you refresh after login it'll prompt logged in info there. I think its something to do with how I'm using client vs server sided stuff. If you have any ideas, let me know, I'll try to solve it.

Reply:
resolved, had to add global state management and wrap the app with it